scene/gui/control.cpp

```diff
@@ -452,10 +453,8 @@ void Control::_notification(int p_notification) {
 
 		} break;
 		case NOTIFICATION_POST_ENTER_TREE: {
```

Comment:
@reduz Can I remove `NOTIFICATION_POST_ENTER_TREE`? this is used only here then I don't know why do you create this. I can move this two lines above to `NOTIFICATION_ENTER_TREE`.

Reply:
See here: https://github.com/godotengine/godot/search?q=NOTIFICATION_POST_ENTER_TREE&unscoped_q=NOTIFICATION_POST_ENTER_TREE

Reply:
No, because computing size with out of tree controls should never happen, and children nodes receive that notification after the parent, so for cases like this it's needed.

Reply:
Ok, makes sense, thanks for explain. :+1: